src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +24,949 @@ to make sure IE does not go into compatibility mode.
+
+&nbsp;
+### Capturing Errors / Exceptions
+
+The most common form of capturing is to capture errors.  What can be captured as an
+error will depend on the platform.  In general if you have something that looks like
+an exception it can be captured.  For some SDKs you can also omit the argument to
+`capture_exception` and it will attempt to capture the current exception.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+&nbsp;
+### Generate a Custom Error
+**[Mimi note: I'd like to flesh this out more, but need help.]**
+
+You can generate a custom error with `new Error('message')`.
+
+&nbsp;
+### Setting Context
+In addition to the [structured context](#context) that Sentry understands, you can send [arbitrary key/value pairs of data](#extra-context) which the Sentry SDK will store alongside the event. These are not indexed, and the Sentry SDK uses them to add additional information about what might be happening.
+
+&nbsp;
+### Capturing Messages
+
+Another common operation is to capture a bare message.  A message is just some textual
+information that should be sent to Sentry.  Typically messages are not emitted but
+there are situations when this is useful.
+
+```
+Sentry.captureMessage('Something went wrong');
+```
+
+&nbsp;
+### Capturing Events
+
+SDKs also provide ways to capture entire custom event objects. This is what integrations internally use to capture bespoke events that the SDK supplies with a lot of extra data.
+
+&nbsp;
+## Source Maps
+
+{% include components/alert.html
+  title="Note"
+  content="We **highly recommend** you incorporate source maps with the Sentry JavaScript SDK in order to receive the full benefit of error tracking and monitoring. Source maps will provide stack traces, which will provide more information regarding errors."
+  level="warning"
+%}
+
+Sentry supports un-minifying JavaScript via source maps. This lets you view source code context obtained from stack traces in their original untransformed form, which is particularly useful for debugging minified code (e.g. UglifyJS), or transpiled code from a higher-level language (e.g. TypeScript, ES6).
+
+When you're using the Sentry JavaScript SDK, the SDK automatically fetches the source code and source maps by scraping the URLs within the stack trace. However, you may have legitimate reasons for [disabling the JavaScript source fetching in Sentry](https://blog.sentry.io/2018/07/17/source-code-fetching).
+
+**[Mimi note: How do you "turn on" Source Maps? This feels like an important question to answer right away. Added Generating Source Maps below.]**
+
+&nbsp;
+### Generating Source Maps
+Most modern JavaScript transpilers support source maps. Below are instructions for some common tools.
+
+&nbsp;
+#### UglifyJS
```

Comment:
Not a blocker now, but we may want to make sections like this collapsible.

Reply:
Not supported out of the box in Readme.io but we could CSS it fwiw: https://feedback.readme.io/feature-requests/p/collapsible-md-sections-or-code-examples